File: app/src/main/java/azalea/com/suchwating/sample/TriangleFragment.java
package azalea.com.suchwating.sample;

import android.graphics.Color;

import com.azalea.suchwaiting.loadingimplemention.BaseLoadingRender;
import com.azalea.suchwaiting.loadingimplemention.triangle.TriangleDotRender;


/**
 * 普通三角形sample
 */
public class TriangleFragment extends FiveLoadRenderFragment {

	@Override
	protected BaseLoadingRender[] getFiveLoadingRender() {
		return new BaseLoadingRender[]{
				new TriangleDotRender(),
				new TriangleDotRender(3000L, Color.BLUE),
				new TriangleDotRender(Color.RED),
				new TriangleDotRender(2000L, Color.parseColor("#00c300")),
				new TriangleDotRender(Color.YELLOW),
		};
	}
}

File: app/src/main/java/azalea/com/suchwating/sample/TriangleFragment.kt
package azalea.com.suchwating.sample

import android.graphics.Color
import com.azalea.suchwaiting.loadingimplemention.BaseLoadingRender
import com.azalea.suchwaiting.loadingimplemention.triangle.TriangleDotRender

/**
 * 普通三角形sample
 */
class TriangleFragment : FiveLoadRenderFragment() {
    override val fiveLoadingRender: Array<BaseLoadingRender> = arrayOf(
            TriangleDotRender(),
            TriangleDotRender(3000L, Color.BLUE),
            TriangleDotRender(1000L, Color.RED),
            TriangleDotRender(400L, Color.parseColor("#00c300")),
            TriangleDotRender(6000L, Color.YELLOW))
}